README.rst

```diff
@@ -1,25 +1,30 @@
-########
-Overview
-########
+|Stable| |Tag| |License| |Build| |Coverage| |Quality|
 
-|License| |Build| |Coverage| |Quality|
+.. raw:: html
 
-.. attention::
+  <div align="center">
+    <h1><code>kytos/flow_manager</code></h1>
 
-    THIS NAPP IS STILL EXPERIMENTAL AND ITS EVENTS, METHODS AND STRUCTURES MAY
-    CHANGE A LOT ON THE NEXT FEW DAYS/WEEKS. USE IT AT YOUR OWN DISCRETION
+    <strong>NApp that manages OpenFlow 1.3 entries</strong>
 
+    <h3><a href="https://kytos-ng.github.io/api/flow_manager.html">OpenAPI Docs</a></h3>
+  </div>
 
-The *kytos/flow_manager* NApp exports a REST API to add, remove and
-list flows from OpenFlow switches, for versions 1.0 and 1.3.
-It can be used by other applications to manage flows with the supported fields.
 
-This application creates an abstraction layer to other applications:
-it is only necessary to know the endpoints. The application handles
-the requests and returns the information already formatted.
+Features
+========
+
+- REST API to create/update/read/delete flows
+- Expose events to create/update/delete flows
+- Store flows in memory and in a permanent storage
+- Consistency check based on FlowStats to ensure that only expected flows are installed 
+- Consistency check ignored flows based on a cookie range and/or table id range
+- Send barrier request and replies
+- Handle southbound socket connection errors as applicable
+- Option to allow to install a flow later on when a switch is connected
 
 Supported Fields
-****************
+================
 
 This NApp supports a subset of the OpenFlow specification fields in the bodies of
 the requests when creating and removing flows:
```

Comment:
In fact, it currently supports much more match fields (if not all). See https://github.com/kytos-ng/of_core/blob/master/flow.py#L326-L336

Reply:
Right @ajoaoff, I've mapped it here https://github.com/kytos-ng/flow_manager/issues/72, also related to that, I recall that matching certain masks we haven't implemented and prioritized yet (based on the use cases we have so far)  https://github.com/kytos-ng/flow_manager/issues/24